extensions/amp-access/amp-access.md

```diff
@@ -482,26 +482,6 @@ This section will cover a detailed explanation of the design underlying the amp-
 [12]: #metering
 [13]: #first-click-free
 
-## Validation errors
+## Validation
 
-The following lists validation errors specific to the `amp-access` tag
-(see also `amp-access` in the [AMP validator specification](https://github.com/ampproject/amphtml/blob/master/extensions/amp-access/0.1/validator-amp-access.protoascii)):
```

Comment:
I understand that you are making the change now that the spec is split into smaller parts, but at the risk of bike shedding, I think the prior phrasing is a bit clearer.

If you want to make the point that the specification is split now, maybe something like "See also amp-access rules in the AMP validator specification".